troubadix/plugins/spelling.py

```diff
@@ -19,19 +19,171 @@
+            "Policy/policy_BSI-TR-03116-4.nasl",
+            "2012/gb_secpod_ssl_ciphers_weak_report.nasl",
+        ],
+        r"bund\s+==>\s+bind",
+    ),
+    # False positive in this VT in German example responses.
+    PatternInFileCheck(
+        "gb_exchange_server_CVE-2021-26855_active.nasl", r"ist\s+==>\s+is"
+    ),
+    # Mostly a false positive in LSCs because of things like
+    # "ALSA: hda" or a codec called "Conexant". There are too
+    # many hits to maintain them in codespell.exclude so exclude
+    # them for now here.
+    PatternInFilePatternCheck(
+        r"gb_(sles|(open)?suse|ubuntu_USN)_.+\.nasl",
+        r"(hda|conexant)\s+==>\s+(had|connexant)",
+        text_pattern_flags=re.IGNORECASE,
+    ),
+    # Jodie Chancel is a security researcher who is mentioned
+    # many times in Mozilla advisories
+    PatternInFilePatternCheck(
+        r"gb_mozilla_firefox_mfsa_\d{4}-\d{2,4}_lin\.nasl",
+        r"Chancel\s+==>\s+Cancel",
+    ),
+    # Look like correct as this is also in dictionary_rare.txt
+    PatternInFileCheck("deb_dla_2896.nasl", r"dependant\s+==>\s+dependent"),
+    # Similar to the one above for e.g. SLES.
+    # Also exclude "tre", because it's a package name.
+    PatternInFilePatternCheck(
+        r"mgasa-\d{4}-\d{4}.nasl",
+        r"(hda|tre|conexant)\s+==>\s+(had|tree|connexant)",
+        file_pattern_flags=re.IGNORECASE,
+    ),
+    # Similar to the corrections above, with some additional
+    # exclusions like e.g. names
+    PatternsInFilePatternCheck(
+        r"ELSA-\d{4}-\d{4,5}\.nasl",
+        [
+            (r"Stange\s+==>\s+Strange", 0),
+            (r"chang\s+==>\s+change, charge", 0),
+            (r"IST\s+==>\s+IS, IT, ITS, IT'S, SIT, LIST", 0),
+            (r"hda\s+==>\s+had", 0),
+            (r"Readded\s+==>\s+Read", 0),
+            (r"ACI\s+==>\s+ACPI", re.IGNORECASE),
+            (r"UE\s+==>\s+USE, DUE", 0),
+        ],
+    ),
+    # "Unsecure" is used in the references so we shouldn't
+    # change that.
+    PatternInFilesCheck(
+        [
+            "office2013_allow_insecure_apps_catalogs.nasl",
+            "gb_sap_rfc_default_pw.nasl",
+            "gb_sap_webgui_default_pw.nasl",
+        ],
+        r"[Uu]nsecure\s+==>\s+[Ii]nsecure",
+    ),
+    # NAM / nam is the abbreviation of these products. In
+    # netop_infopublic.nasl there is a "nam" parameter.
+    PatternInFilePatternCheck(
+        r"gb_((cisco|solarwinds)_nam|netiq_access_manager)_",
+        r"nam\s+==>\s+name",
+        text_pattern_flags=re.IGNORECASE,
+    ),
+    PatternInFileCheck(
+        "/netop_infopublic.nasl", r"nam\s+==>\s+name", flags=re.IGNORECASE
+    ),
+]
+
+
 class CheckSpelling(FilesPlugin):
     name = "check_spelling"
 
+    def _parse_codespell_line(self, line: str) -> Tuple[str, str]:
+        if not "==>" in line:
+            raise ValueError("Invalid codespell line")
+
+        file, _, correction = line.split(":")
+
+        return file, correction
```

Comment:
```suggestion
        return file, correction.strip()
```
Otherwise `correction` contains e.g. `" faeture ==> feature'"`, which doesn't break anything, but is a bit unclean IMHO

Reply:
Done